spellbook/models/tokens/avalanche/tokens_avalanche_schema.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+
+models:
+  - name: tokens_avalanche_erc20
+    meta:
+      blockchain: avalanche
+      sector: tokens
+      project: erc20
+      contibutors: 0xManny
+    config:
+      tags: ['table', 'erc20', 'avalanche']
+    description: "ERC20 Token Addresses, Symbols and Decimals"
+    columns:
+      - name: contract_address
+        description: "ERC20 token contract address on avalanche"

```

Comment:
please add unique test to the contract address, since this is manual file and dupes can be easily added.

example:
abstractions/spellbook/models/tokens/ethereum/tokens_ethereum_schema.yml